centaur/src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -42,6 +42,7 @@ object TestFormulas extends StrictLogging {
     _ <- checkDescription(workflowDefinition, validityExpectation = Option(true))
     submittedWorkflow <- runSuccessfulWorkflow(workflowDefinition)
     metadata <- validateMetadata(submittedWorkflow, workflowDefinition)
+    _ <- validateJobManagerStyleMetadata(submittedWorkflow, metadata.value)
```

Comment:
Any other places this should go?

Reply:
I *think* it could go anywhere `runSuccessfulWorkflow` is being called?

Reply:
In fact, every time we record the metadata, we can check that it gets JM-ified properly.